### PR TITLE
fix regex for persian month -->>the "دی" month has only 2 characters …

### PR DESCRIPTION
**Pull request by javadfarah**
fix regex for Persian month -->>the "دی" month has only 2 characters ……and the minimum of this regex group should start from 2 chars

### CODE REVIEW

**Comment by javadfarah** [comment]
fix regex for Persian month -->>the "دی" month has only 2 characters …

**Comment by hramezani** [comment]
Thanks @javadfarah for this patch :+1: 
Could you please add a test for this? You can add your test in [test_strptime_handle_b_B_directive](https://github.com/slashmili/python-jalali/blob/ff9c2e1cea56da2fd084a2c9482c851c5415044d/tests/test_jdatetime.py#L360)

**Comment by javadfarah** [comment]
@hramezani sure, Thanks for the quick reply, I already added  tests and  fixed pep8 violations in the test file

File: tests/test_jdatetime.py [hunk 1]
```diff
@@ -385,6 +386,8 @@ def test_strptime_handle_b_B_directive(self):
             with self.subTest(date_string=date_string, date_format=date_format):
                 date = jdatetime.datetime.strptime(date_string, date_format)
                 self.assertEqual(jdatetime.datetime(1400, 2, 14), date)
+        two_chars_month = jdatetime.datetime.strptime('14 دی 1400', '%d %B %Y')
```
Comment (hramezani) [inline tests/test_jdatetime.py:389]: hmmm, seems all the tests listed above are for `14 Ordibehesht 1400`. Please add a new test and move it to the new one. something like `test_strptime_handle_b_B_directive_two_chars_month`

File: tests/test_jdatetime.py [hunk 2]
```diff
@@ -81,8 +82,8 @@ def test_fold(self):
 
         # Test invalid value for fold
         with self.assertRaises(
-            ValueError,
-            msg='fold must be either 0 or 1'
+                ValueError,
```
Comment (hramezani) [inline tests/test_jdatetime.py:85]: Why did you add another ident here?

File: tests/test_jdatetime.py (hunk 1 above)
Comment (javadfarah) [reply tests/test_jdatetime.py]: yes, it makes sense now in the separate test method.

**Comment by hramezani** [comment]
Thanks @javadfarah for the update.
I added a commit to include the new test cases in the existing test and revert the formating changes. we are not using `black` for now.
Now it look good.

**Comment by hramezani** [comment]
Changelog added!

**Review by hramezani — APPROVED** [review]
